src/Indice.Features.Cases.AspNetCore/Services/MyCaseService.cs

```diff
@@ -175,8 +178,8 @@ internal class MyCaseService : BaseCaseService, IMyCaseService
                                          Id = c.Id,
                                          Created = c.CreatedBy.When,
                                          CaseTypeCode = c.CaseType.Code,
-                                         Status = c.PublicCheckpoint.CheckpointType.Status,
-                                         Checkpoint = c.PublicCheckpoint.CheckpointType.Code,
+                                         Status = c.PublicCheckpoint == null ? CaseStatus.InProgress : c.PublicCheckpoint.CheckpointType.Status,
```

Comment:
Δεν ισχύει το αμα είναι null τότε είναι in progress. Βασικά πως γίνεται να είναι null?  

Πάντα στην [create draft internal](https://github.com/indice-co/Indice.AspNet/blob/1dd81eb49595c7af7fa356a3857e446ef1fe80d1/src/Indice.Features.Cases.AspNetCore/Services/BaseCaseService.cs#L162) έχουμε 
```cs
await caseMessageService.Send(entity.Id, user, new Message { CheckpointTypeName = "Submitted" });
```
άρα θα πρέπει να σου φέρνει status = submitted!

Αν δεν στο φέρνει, τότε γι' αυτο το case type πρέπει checkpoint type submitted να έχει το property [private](https://github.com/indice-co/Indice.AspNet/blob/1dd81eb49595c7af7fa356a3857e446ef1fe80d1/src/Indice.Features.Cases.AspNetCore/Data/Models/DbCheckpointType.cs#L13) = false

Reply:
ναι έχεις δίκιο. είχα λάθος data στη βάση και έσκαγε. το γύριζω στην άρχική του μορφή χωρίς null checks